docs/book/cookbook/application-wide-layout.md

```diff
@@ -0,0 +1,85 @@
+# Set Application-Wide Layout
+
+The format for the view helper `FlashMessenger` can be configured for an entire application and all uses of the helper.
```

Comment:
I'd rewrite this as "The format for the `FlashMessenger` view helper can be configured for an entire application as well as all uses of the helper.".